src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -364,7 +366,6 @@ public void insertRecordToBuffer(SinkRecord kafkaSinkRecord) {
       } finally {
         bufferLock.unlock();
       }
-
```

Comment:
is this intended?

Reply:
nope, added new line back in, thanks for the catch